code/models/country.js

```diff
@@ -11,6 +13,7 @@ const countrySchema = new mongoose.Schema({
     map: String, // Same as above 
     locationID: { type:mongoose.Schema.Types.ObjectId, ref: 'Location' }
   },
+  organizations: [OrganizationSchema]
```

Comment:
This isn't how foreign keys are assigned. Check out the trip.js model's locationID. Also, change the name to organizationIDs